.gitignore

```diff
@@ -37,4 +37,6 @@ vendor
 linky-clicky-all-the-things.sh
 uatd6
 */behat.local.yml
+/products/*/behat.local.yml
```

Comment:
We should just change this to one entry for all behat.local.yml files.

Replace the entries in this file with just one ignore.